index.html

```diff
@@ -1182,9 +1182,14 @@ <h2>
         </li>
 
         <li>
-Each public key MUST include <code>id</code> and <code>type</code>
-properties, and exactly one value property. The array of public keys
-MUST NOT contain duplicate entries with the same <code>id</code>.
+Each public key SHOULD include an <code>id</code> property. The array
+of public keys MUST NOT contain duplicate entries with the same
```

Comment:
```suggestion
of public keys MUST NOT contain multiple entries with the same
```
... because *duplicate* would be redundant, not conflicting.